optimize_sidebars.js

```diff
@@ -0,0 +1,1053 @@
+            docsComponentsLink(
+              "Camunda Forms reference",
+              "components/modeler/forms/camunda-forms-reference/"
+            ),
+          ],
+        },
+      ],
+    },
+
+    {
+      Zeebe: [
+        docsComponentsLink("Introduction", "components/zeebe/zeebe-overview/"),
+
+        {
+          "Technical concepts": [
+            docsComponentsLink(
+              "Overview",
+              "components/zeebe/technical-concepts/technical-concepts-overview/"
+            ),
+            docsComponentsLink(
+              "Architecture",
+              "components/zeebe/technical-concepts/architecture/"
+            ),
+            docsComponentsLink(
+              "Clustering",
+              "components/zeebe/technical-concepts/clustering/"
+            ),
+            docsComponentsLink(
+              "Partitions",
+              "components/zeebe/technical-concepts/partitions/"
+            ),
+            docsComponentsLink(
+              "Internal processing",
+              "components/zeebe/technical-concepts/internal-processing/"
+            ),
+            docsComponentsLink(
+              "Process lifecycles",
+              "components/zeebe/technical-concepts/process-lifecycles/"
+            ),
+            docsComponentsLink(
+              "Protocols",
+              "components/zeebe/technical-concepts/protocols/"
+            ),
+          ],
+        },
+
+        {
+          "Open source community": [
+            docsComponentsLink(
+              "Community contributions",
+              "components/zeebe/open-source/community-contributions/"
+            ),
+            docsComponentsLink(
+              "Get help and get involved",
+              "components/zeebe/open-source/get-help-get-involved/"
+            ),
+          ],
+        },
+      ],
+    },
+
+    {
+      Operate: [
+        docsComponentsLink(
+          "Introduction",
+          "components/operate/operate-introduction/"
+        ),
+
+        {
+          "User guide": [
+            docsComponentsLink(
+              "Getting familiar with Operate",
+              "components/operate/userguide/basic-operate-navigation/"
+            ),
+            docsComponentsLink(
+              "Variables and incidents",
+              "components/operate/userguide/resolve-incidents-update-variables/"
+            ),
+            docsComponentsLink(
+              "Selections and operations",
+              "components/operate/userguide/selections-operations/"
+            ),
+            docsComponentsLink(
+              "Delete finished instances",
+              "components/operate/userguide/delete-finished-instances/"
+            ),
+            docsComponentsLink(
+              "Giving feedback and asking questions",
+              "components/operate/userguide/operate-feedback-and-questions/"
+            ),
+          ],
+        },
+      ],
+    },
+
+    {
+      Optimize: [
+        "what-is-optimize",
```

Comment:
This is the only link in this sidebar file that is doc-based instead of URL based (because it's the only doc moved over in this PR).